apps/app/src/components/rss-feed/RssFeedItem.tsx

```
@@ -0,0 +1,79 @@
+import { Badge } from "@/components/ui/badge";
+import { Card, CardContent } from "@/components/ui/card";
+import type { RssFeedItem } from "@/types/rss";
+
+interface RssFeedItemProps {
+  item: RssFeedItem;
+  onCategoryClick: (category: string) => void;
+}
+
+export function RssFeedItem({ item, onCategoryClick }: RssFeedItemProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'RssFeedItem'. Consider to delete it or rename it.

'RssFeedItem' is defined here:


(lint/suspicious/noRedeclare)